pulsar-common/src/main/java/org/apache/pulsar/common/util/SecurityUtility.java

```diff
@@ -115,7 +114,7 @@ public static Provider getProvider() {
         }
     }
 
-    private static Provider loadConscryptProvider() {
+    public static Provider loadConscryptProvider() {
```

Comment:
this shouldn't be exposed. This method is meant to be called exactly once. Instead, `if(SecurityUtility.CONSCRYPT_PROVIDER != null) {` could be used in some other class to see if Conscrypt is available.

Reply:
@lhotari Good point! This has been fixed.